api-report/sequence.api.md

```diff
@@ -78,10 +78,10 @@ export function createIdIntervalIndex<TInterval extends ISerializableInterval>()
 export function createOverlappingIntervalsIndex<TInterval extends ISerializableInterval>(client: Client, helpers: IIntervalHelpers<TInterval>): IOverlappingIntervalsIndex<TInterval>;
 
 // @public (undocumented)
-export function createOverlappingSequenceIntervalsIndex(client: Client): SequenceIntervalIndexes.Overlapping;
+export function createOverlappingSequenceIntervalsIndex(sharedString: SharedString): SequenceIntervalIndexes.Overlapping;
 
 // @public (undocumented)
-export function createStartpointInRangeIndex<TInterval extends ISerializableInterval>(helpers: IIntervalHelpers<TInterval>, client: Client): IStartpointInRangeIndex<TInterval>;
+export function createStartpointInRangeIndex<TInterval extends ISerializableInterval>(client: Client, helpers: IIntervalHelpers<TInterval>): IStartpointInRangeIndex<TInterval>;
```

Comment:
this should take in same parameters as `createStartpointInRangeIndex`

Reply:
yeah I will update it (as well as its unit tests) after we approve the latest commit about `createEndpointInRangeIndex`